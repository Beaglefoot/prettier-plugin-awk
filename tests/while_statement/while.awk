{
  while(1) {
    print 42  }
  do print 42
  while(1)
}
