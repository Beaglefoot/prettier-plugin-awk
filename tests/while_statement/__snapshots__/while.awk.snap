{
  while(x<2) {
    print 42  }
  do print 42
  while(1)
}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
{
    while (x < 2) { print 42 }

    do print 42
    while (1)
}
